test/AmrPpm/SConscript

```diff
@@ -26,9 +26,9 @@ env.Append(BUILDERS = { 'pngToGif'  : png_to_gif})
 
 
 
-run_adapt = Builder(action = "$RMIN; " + date_cmd + parallel_run + " $SOURCE $ARGS " + " > $TARGET 2>&1; $CPIN; $COPY")
+run_adapt = Builder(action = "$RMIN; " + date_cmd + parallel_run + " $SOURCE $clocal_cmd  $ARGS " + " > $TARGET 2>&1; $CPIN; $COPY")
 env.Append(BUILDERS = { 'RunAdapt' : run_adapt } )
-env_mv_adapt = env.Clone(COPY = 'mkdir -p ' + test_path + '/AmrPpm/Adapt-L5-P1; mv `ls *.png *.h5` ' + test_path + '/AmrPpm/Adapt-L5-P1')
```

Comment:
I just wanted to confirm that the removal of the second part of this command is intentional. 

Lot's of other environments seem to have that second part of the command (like `env_mv_heat_8` from `test/MethodHeat/SConscript`).

Reply:
I've put this back in now